js/models/ShippingAddress.js

```diff
@@ -0,0 +1,16 @@
+import BaseModel from './BaseModel';
+
+export default class extends BaseModel {
+  defaults() {
+    return {
+      Name: '',
+      Company: '',
+      AddressLineOne: '',
+      AddressLineTwo: '',
+      City: '',
+      State: '',
+      Country: '',
+      PostalCode: '',
```

Comment:
These keys are going to change. That's not an error, but we should discuss what we're going to use for the v2 address fields.